libs/infra-tracing/src/lib/datadog-tracer.ts

```diff
@@ -1,8 +1,8 @@
-import tracer from 'dd-trace'
+// import tracer from 'dd-trace'
```

Comment:
commented?

Reply:
aw shuck.... forgot this.

there were some huge problems with dd-trace when I started wrt dynamic imports. I'll take another stab at it.

Reply:
Fixed via https://github.com/island-is/island.is/pull/3999